pkg/util/log.go

```diff
@@ -0,0 +1,183 @@
+package util
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/klog"
+)
+
+type Verbosity struct {
+	level int
+}
+
+func V(i int) *Verbosity {
+	return &Verbosity{level: i}
+}
+
+type contextKey string
+
+var Key = contextKey("ID")
```

Comment:
do we need to expose this var?

Reply:
yes is is used [here](https://github.com/ceph/ceph-csi/blob/7dedcda5fdb3a67218db1f66245da786a397589d/pkg/csi-common/utils.go#L114)